src/main/resources/descriptions/holiday_descriptions_en.properties

```diff
@@ -21,6 +21,7 @@ holiday.description.BATTLE_BOYNE                   = Battle of the Boyne
 holiday.description.BATTLE_JACINTO                 = Battle of San Jacinto
 holiday.description.BEER_DAY                       = Beer Day
 holiday.description.BENNINGTON                     = Battle of Bennington
+holiday.description.BIRTHDAY_OF_KING               = Youth Day
```

Comment:
The key ("BIRTHDAY_OF_KING") and the description "Youth Day" are not really in sync, perhaps you should find a better key?

Also, BIRTHDAY_OF_KING ... Many countries have kings, so maybe call the key BIRTHDAY_OF_MAROCCAN_KING ?

Reply:
You are right, here in morocco the two notations refer to the same holiday, because the youth day is fixed on the king's birthday.
Thanks